MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+        <DataTemplate>
+            <ViewCell>
+                <ViewCell.View>
+                    <Grid>
+                        <local:ExtendedListView x:Name="list1" HeightRequest="{Binding InnerListHeight}" ItemsSource="{Binding ContactDetails}" ItemSize="75">
+                            <local:ExtendedListView.ItemTemplate>
+                                <DataTemplate>
+                                    <StackLayout>
+                                        <Image Source="{Binding Image}" />
+                                        <Label Text="{Binding Name}" />
+                                        <Label Text="{Binding Number}" />
+                                    </StackLayout>
+                                </DataTemplate>
+                            </local:ExtendedListView.ItemTemplate>
+                        </local:ExtendedListView>
+                    </Grid>
+                </ViewCell.View>
+            </ViewCell>
+        </DataTemplate>
+    </listView:SfListView.ItemTemplate>
+</listView:SfListView>
+{% endhighlight %}
+{% highlight C# %}
+ public partial class NestedListView : ContentPage
+{
+    public NestedListView()
+    {
+        InitializeComponent();
+    }
+
+    private async void listView_Loaded(object sender, Syncfusion.Maui.ListView.ListViewLoadedEventArgs e)
+    {
+        await Task.Delay(2000);
+        listView.RefreshView();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+{% highlight C# %}
+using Syncfusion.Maui.ListView.Control.Helpers;
+public class ExtendedListView : SfListView
+{
+    VisualContainer container;
+    public ExtendedListView()
+    {
+        container = this.GetVisualContainer();
+        container.PropertyChanged += Container_PropertyChanged;
+    }
+    
+    private void Container_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+    {
+        Device.BeginInvokeOnMainThread(async () =>
+        {
+            var extent = (double)container.GetType().GetRuntimeProperties().FirstOrDefault(container => container.Name == "TotalExtent").GetValue(container);
+            if (e.PropertyName == "Height")
+                (this.BindingContext as ContactInfo_NestedListView).InnerListHeight = extent;
+        });
+    }
+}
+{% endhighlight %}
+
+ 
+## Rendering ListView when loading in different layouts
+
+The options are as follows:
+
+* Creates the measurement and layout similar to .NET MAUI ListView, when the ListView is loaded inside the layouts such as `StackLayout`, `ScrollView`, and `Grid`, in which the `RowDefinition` or `ColumnDefinition` is set to `Auto`. In all other layouts, the ListView size will be allocated from the framework.
+* Sets the value of total extend to the `HeightRequest` of ListView, since the scrolling will be handled by the parent `ScrollView`, when ListView is loaded inside the `StackLayout` with base parent as `ScrollView` having multiple elements inside the `StackLayout`.

```

Comment:
Sets the value of total extend to the `HeightRequest` of ListView, because the scrolling will be handled by the parent `ScrollView`, when ListView is loaded inside the `StackLayout` with base parent as `ScrollView` and multiple elements inside the `StackLayout`.

Reply:
I have modified the line as per the concerns provided